CHANGELOG_PENDING.md

```diff
@@ -4,5 +4,7 @@
 * `aws/session`: Ignore invalid shared config file when not used ([#2731](https://github.com/aws/aws-sdk-go/pull/2731))
   * Updates the Session to not fail to load when credentials are provided via the environment variable, the AWS_PROFILE/Option.Profile have not been specified, and the shared config has not been enabled.
   * Fixes [#2455](https://github.com/aws/aws-sdk-go/issues/2455)
+* `aws/session`: Add support for CSM options from shared config file ([#]())
```

Comment:
Need the PR number [#2768 ] and link `https://github.com/aws/aws-sdk-go/pull/2768`

Reply:
Thanks! missed that